changelogs/unreleased/5306-improved-attribute-hash.yml

```diff
@@ -0,0 +1,9 @@
+description: Improve stability of incremental deploy for resources containing dicts
+issue-nr: 5306
+sections:
+  feature: "{{description}"
+  upgrade-note: "The first recompile after this upgrade will always perform a full deploy"
+change-type: minor
+destination-branches:
+  - iso6
+  - master
```

Comment:
Shouldn't this also go into iso5. iso5 has partial compile as well

Reply:
This is a fairly intrusive fix for a fairly exotic problem, so I would say no.